staging/src/k8s.io/apimachinery/pkg/watch/mux_test.go

```diff
@@ -58,6 +59,7 @@ func TestBroadcaster(t *testing.T) {
 	wg := sync.WaitGroup{}
 	wg.Add(testWatchers)
 	for i := 0; i < testWatchers; i++ {
+		w, _ := m.Watch()
```

Comment:
nit: can we add error handling in all places that miss it now?

Reply:
Yep I will add